src/assets/styles/_mixins.scss

```diff
@@ -1,5 +1,115 @@
 // Mixins
 
+@mixin calciteLightThemeVars {
```

Comment:
Sorry if we discussed this, but I am not clear on why these vars are in a mixin. Same with the DarkThemeVars.

Reply:
The light vars are stated twice so its in a mixin for that